src/actions/render.js

```diff
@@ -1,25 +1,31 @@
-const { groupByKey } = require("../utils/arrayUtils");
-const { mapObject } = require("../utils/objectUtils");
+const Mustache = require("mustache");
+const Result = require("../utils/result");
```

Comment:
In other files, I grouped imports into external dependencies and internal dependencies. Following that logic, `Result` should be separated at least by a newline.

It's probably best to leave this to the linter by using [something like this](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)?